examples/tutorials/VaraibilityModelExamples.ipynb

```diff
@@ -0,0 +1,302 @@
+   "outputs": [],
+   "source": [
+    "import numpy\n",
+    "from lsst.sims.catalogs.measures.instance import InstanceCatalog\n",
+    "from lsst.sims.catUtils.mixins import PhotometryStars"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "\n",
+    "class uMagBaselineClass(InstanceCatalog, PhotometryStars):\n",
+    "    column_outputs = ['uniqueId', 'raJ2000', 'decJ2000', 'lsst_u']\n",
+    "    transformations = {'raJ2000':numpy.degrees, 'decJ2000':numpy.degrees}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from lsst.sims.utils import ObservationMetaData"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "#specify the telescope pointing\n",
+    "obs = ObservationMetaData(unrefractedRA=0.0, unrefractedDec=0.0,\n",
+    "                          boundType='circle', boundLength=2.0,\n",
+    "                          mjd=52350.0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "baselineCat = uMagBaselineClass(rrlyDB, obs_metadata=obs)\n",
+    "baselineCat.write_catalog('baseline_catalog.txt')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we will print out our catalog to the notebook"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "!cat baseline_catalog.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Variability is controlled by the mixins `VariabilityStars` and `VariabilityGalaxies`.  These mixins calculate columns `delta_lsst_u`, `delta_lsst_g`, `delta_lsst_r`, etc. and automatically add them to the baseline magnitude columns.\n",
+    "\n",
+    "Below, we take the same telescope pointing, but turn on variability and calculate the magnitudes.  Compare the output below to the output frome the baseline catalog above.  Note that we have output the `delta_lsst_u` column as well for comparison."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from lsst.sims.catUtils.mixins import VariabilityStars"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "class uMagVarClass(InstanceCatalog, PhotometryStars, VariabilityStars):\n",
+    "    column_outputs = ['uniqueId', \n",
+    "                      'raJ2000', 'decJ2000', \n",
+    "                      'lsst_u', 'delta_lsst_u']"
+   ]
```

Comment:
Include Transformations to degrees so that viewers can just focus on change in magnitudes rather than thinking at first glance everything has changed when printing out catalog below?